package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "vscode-aks-tools",
-    "version": "0.0.17",
+    "version": "0.0.18",
```

Comment:
Should this change have happened as part of another update?

Reply:
👍 you are correct, so when we create/write publish PR along with readme change et al.